src/gsy_e/gsy_e_core/simulation.py

```diff
@@ -99,7 +99,7 @@ def __init__(self, setup_module_name: str, simulation_config: SimulationConfig,
 
         self.setup_module_name = setup_module_name
         self.is_stopped = False
-
+        self._incremental_slots = incremental
```

Comment:
Nit: At first glance, because of the naming, I would expect this variable to contain slots (not a boolean). Why not name it `incremental`? Anyway, not a big deal.

Reply:
Well both comments make sense, I missed the second one, let find a common ground

Reply:
What do you have on `self._increment_slots` ?

Reply:
I'd suggest keeping it as it is now so that we can merge it. Any name might sound "incomplete" somehow, so let's give up to language ambiguity :)